src/cms/page-builder-elements-buttons.md

```diff
@@ -1,29 +1,57 @@
 ---
 ee_only: true
-title: Buttons
+title: Elements - Buttons
 ---
 
 Use the Buttons content type to add either an individual button or a set of buttons to the stage. Buttons can be arranged horizontally or vertically, and added directly to rows, columns, tabs, and banners on the stage.
 
 ![]({% link images/images-ee/page-builder-storefont-banner-with-button.png %}){: .zoom}
-_Banner with Button_
+_Banner with a button on the storefront_
 
-## To add an individual button
+## Toolboxes
 
-1. In the Page Builder panel under **Elements**, drag a **Buttons** placeholder to a row, column, or tab set on the stage.
+When you are working with the Buttons content type, you add and edit individual buttons and the buttons container that holds one or more buttons. Each of these has its own toolbox that you use to design buttons on the Page Builder stage.
 
-   ![]({% link images/images-ee/page-builder-elements-button-drag.png %}){: .zoom}
-   _Drag Button to Stage_
+### Individual button toolbox
+
+![]({% link images/images-ee/page-builder-elements-button-settings.png %}){: .zoom}
+_Button toolbox_
+
+| Tool      | Icon     | Description    |
+| --------- | -------- | -------------- |
+| Settings  | ![]({% link images/images-ee/icon-pb-settings.png %})  | Opens the Edit Button page, where you can change the properties of the button. |
+| Duplicate | ![]({% link images/images-ee/icon-pb-duplicate.png %}) | Makes a duplicate copy of the button. |
+| Remove    | ![]({% link images/images-ee/icon-pb-remove.png %})    | Deletes the button from the stage. |
+
+### Buttons container toolbox
+
+![]({% link images/images-ee/page-builder-elements-buttons-toolbox-settings.png %}){: .zoom}
+_Buttons container toolbox_
+
+| Tool      | Icon              | Description |
+| --------- | ----------------- | ----------- |
+| Move      | ![]({% link images/images-ee/icon-pb-move.png %}) | Moves the button container to another valid place on the page. |
+| Add       | ![]({% link images/images-ee/icon-pb-add-button.png %}) | Adds a new button to the container. |
+| (label)   | Button  | Identifies the current container as a button element. |
```

Comment:
That is because it is a label to identify the toolbox. I thought about taking these out, but decided to keep them to help when there is more than one toolbox involved with a content type.